src/pilot/dell_nfv.py

```diff
@@ -55,50 +57,6 @@ def __init__(self, overcloud_name):
         self.overcloudrc = "source " + home_dir + "/"\
             + self.overcloud_name + "rc;"
 
-    def reboot_compute_nodes(self):
```

Comment:
I would have preferred to keep this one as it's easier to track and log the reboot. Besides, logging these reboot will help us to make sure it has been done successfully.

Reply:
I am not sure about it.
In master code we rebooting twice in case dpdk is enable. (In first-boot.yaml and Post deployment)
If we remove reboot from first-boot-ovs-dpdk.yaml then dpdk won't work as NUMA and Hugepages are prerequisites for DPDK.
We did same mistake earlier by removing NUMA HUGPAGES code from first-boot-ovs-dpdk.yaml and it didn't work.
So we have to keep the reboot and also we don't want to reboot twice.

Reply:
yes i just figure that out. This reboot is done after deployment as opposed to your reboot which is done after the first boot but before Puppet does his job. Hence your failure.